app/bundles/LeadBundle/Entity/LeadRepository.php

```diff
@@ -578,6 +578,34 @@ public function getEntities($args = array())
             ) : $results;
     }
 
+
+    public function getEntitiesLight($args = array()){
+    	$limit = 20000; //lines
+    	$results = array();
+    	$offset = 0;
+    	$result = $this->requestEntitiesLight($args, $offset, $limit);
+    	$results = array_merge($results, $result);
+    	while (count($result) === $limit){
+    		$offset += $limit;
+    		$result = $this->requestEntitiesLight($args, $offset+1, $limit);
```

Comment:
Here is also some wrong indentation. Also, there should be a space before the `{` and the opening `{` of the method should be on a new line. See http://symfony.com/doc/current/contributing/code/standards.html for more details.